src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,67 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+import { PostInfo } from '../PostInfo';
+import { Post } from '../../types/Post';
+import { User } from '../../types/User';
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  posts: Post[],
+  comments: Comment[],
+  users: User[],
+};
+
+function createPostsInfo(
```

Comment:
It would be better to create this function in `App` component and pass to the `PostList` only array of posts, instead of `posts, users, comments`